README.md

```diff
@@ -7,3 +8,129 @@
 A dataframe auditor that computes a number characteristics of the data. This is done by profiling the frame.
 
 
+
+> [Summary](#summary)
+> 
+> [Installation](#installation)
+>
+> [Testing](#testing)
+>
+> [Usage](#usage)
+> 
+> [Contributions](#contributions)
+
+## Summary
+
+  Data profiling is important in data analysis and analytics, as well as in determining characteristics of data pipelines.
```

Comment:
[Data profiling](https://en.wikipedia.org/wiki/Data_profiling)

Reply:
sorry I meant you should put url/link like i did perhaps

Reply:
ah. cool cool cool. no doubt no doubt

Reply:
Added link. interestingly, we are using it in more continual-validation style approach, but yeah, that link does speak to the roots.